File: submissions/sm_027_rudeshwar/week_03/day_4/session_1/mirrorImage.js
// WRITE A FUNCTION THAT ACCEPTS A STRING
// CREATE A VARIABLE THAT STORES INFORMATION OF ALL ALPHABETS IN CAPITAL AND SMALL LETTER
// INFO = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
// A will have an index of 26
// a will have an index of 0
// CREATE A EMPTY STRING OUTPUT
// FOR EVERY ELEMENT IN THE INPUT STRING:
//     IF INFO.INDEXOF(ELEMENT) IS VALID:
//         IF VALUE IS LESS THAN 26:
//             IF VALUE IS LESS THAN 13:
//                 OUTPUT += INFO[25 - VALUE]
//                ELSE:
//                 OUTPUT += INFO[12 - VALUE%13]
//         ELSE:
//             IF VALUE IS LESSER THAN 25+13:
//                 OUTPUT += INFO[51 - VALUE%26]
//             ELSE:
//                 OUTPUT += INFO[26 + VALUE%(26+13)]
// RETURN OUTPUT
var str1="AyzB"
// var b="ZbaY"
var VALUE
var OUTPUT=" "
var INFO="abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
function mirrorImage(str1)
{   
    for(i=0;i<=str1.length-1;i++)
    {   
        if(INFO.indexOf(str1)<=51)
        {
            if(VALUE<26)
            {
                if(VALUE<13)
                {
                    console.log(OUTPUT += INFO[25 - VALUE])
                }
                else
                {
                    console.log(OUTPUT += INFO[12 - VALUE%13])
                }
            }
            else
            {
                if(VALUE<38)
                {
                console.log(OUTPUT += INFO[51 - VALUE%26])
                }
                else
                {
                console.log(OUTPUT += INFO[26 + VALUE%(39)])
                }
            }
        }
        else
        {
            return -1
        }
    }
        return OUTPUT
}
ans=mirrorImage(str1)//passing str1 to function
console.log(ans)

File: submissions/sm_021_piyush-jain/week_13/day_4/mirror_image.py
upper=["A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z"]
lower=["a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z"]
string="ZbaY"
ans=""
for x in string:
    if(x) in upper:
        index=upper.index(x)
        ans=ans+str(upper[25-index])
    else:
        index=lower.index(x)
        ans=ans+str(lower[25-index])
print(ans)